demo.exp

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/expect -f
```

Comment:
Is including this file intentional?

Reply:
It is for local testing. I have removed it.